tests/resources/piwik-1.13-dump.sql

```diff
@@ -70,7 +70,7 @@ CREATE TABLE `archive_blob_2012_03` (
e',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�MN�\n�@��ܕf��M��U�{I�W\nU�쿛�V�0	��L\"dh�T���\\���I>r�.��6�@H�[��Be�a�� H��JT3���ܴ���\'�@E�8(*�^\'�1i�p��q\rM��;�n\\�����Q6S:dq�v�-n��f�����D�'),(5,'Referers_urlByWebsite_1392',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�%MM�0�+dw���v1^<�1�I�*�[X�)�w��{}m�P�$!W�`79�OsF�l5�ad�ar�X�B��#�cZSؒb\"�`z�\rd��Y�i7�������A���C��Q�>q�����1!��e�.`+�ˬb^��E|�����	!�i��1���Q��mD�'),(5,'UserCountry_city',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�}N��@���H��uVf@��J�ÉJ�ԡ�\'镕�g�b�Q��\\�-�����:�C�q��X\'{�I�(�]��נQ�a$�P�H\"�p2T��Ѹ6pبq��E��[����9X�\n�\\�����뼺��6�ޔ����f>��/�%H�'),(5,'UserCountry_country',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�}N�\n�0��܅�ݦK��U���tB� l���n�Ϋ���h�ˋ��5P�g�����q��i���;��/v!��!�2�x�M��\r^qP����r��֡Rk�`��	�[���o�4];N��X�S*uV�Y(��7��9Y��;4��,Iy'),(5,'UserCountry_region',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�}N��@���H��uVf@��J�ÉJ�ԡ�\'镕�g�b�Q��\\�-�����:�C�q��X\'{�I�(�]��נQ�a$�P�H\"�p2T��Ѹ6pبq��E��[����9X�\n�\\�����뼺��6�ޔ����f>��/�%H�'),(5,'UserSettings_browser',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-���0C��vP/�i�# �^]�CD$�\"u���\\�ó-�V8�F.�#�\Z��\ZO�ѻNio��ȋ	=H�?W��mAI�9�,�a�$n��q�\':�����,�ht۔�W�*����Y���k�1@'),(5,'UserSettings_browserType',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-��\n�0����B�i~�[;��t��C�2�{�i�OBR8��J����0W��K���i5��DQ�犾�\r(��#ɞ?~��\0wP�q�7:���ֱY[��S��ER.��d���*�1�'),(5,'UserSettings_configuration',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-��\n�@��e{!{��s���\ZD�.l4�a �B�#��ff�Y�C�(�ރ���gз^t���</Fz�НDQ�犐�\Z4�0N$���z�m�W}|H\"�Fi���ymp�ج���۔�W�YҚߏ���8�/\0�3�'),(5,'UserSettings_language',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-��\n�@��%w���\'W�*⽤ZaqAh��}w���I9��3�\0��5���~��c���{%0�bBwE��+zoP�qJ$�=��L6j�7��@g��6k=\Z�6e��EQ!??I1߱�/tt0�'),(5,'UserSettings_os',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-���@��%;Rs�?8+3T	�*�\'Nb(R�ӽ;	t�l�q9��3�\0\Z�\Z_�Q?z�)���y1�������\r(�4\'��.t�$a�6zc/Ot��m`�֣�mS�_Q\\r����{���0�'),(5,'UserSettings_plugin',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x����j1�w�{a�U����H�2�:nl4ٺ��;�ŋ0�0L��1!!4ς��D3C��A��z�����\"��H�@����	��;F�b.��58A�n�\n�E�h�³V��p�[�uL뵧�}����F;:Q�VЦ�Dn�3�sT@���_i�^��A���B�L���s朔@-� ?mҞ[����k�0Ŕ�MKL��$����f�g���D!t����\\�Z[>'),(5,'UserSettings_resolution',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-��\n�0D�e�B6m�8�zV�e�=Z���n��733��`�|]Ӗ^�IV�˔�������Ѓ������(�4g���������)A��^}P�YVs-*~�2���D`Pj��cU?p_gd1�'),(5,'UserSettings_wideScreen',1,'2012-03-01','2012-03-31',3,'2013-10-08 21:55:44','x�-���0D��;R�Ҵ\\Vf@��r�CD�R��!ʿcC�ww>�9�\ng�t�k|�G��M�4��i%0�Bw\n��?t�6�$Ø(,��4�D��Q+�����ޱ����۔�WA�\\�����-|q�2k'),(5,'VisitorInterest_daysSinceLastVisit',1,'2012-03-01','2012-03-31',3,'2013-10-08
```

Comment:
Those changes should reverted. Same applies to all changes in `tests/resources/OmniFixture-dump.sql`

Reply:
Done